### PR TITLE
[CD-396] Initial port to public repo

### PR DESCRIPTION
**Pull request by nckswt**
## Summary

Public port of a private plugin. We need to make this public so we can use it on Grafana Cloud.

## TODOs

After merge
- [ ] Submit plugin for review
- [ ] Sign plugin

### CODE REVIEW

**Review by JosephRivera10 — APPROVED** [review]
Gleaned over the infrastructure items because of time (config, webpack, etc).
Could use some more context on the grafana interfaces, but not seeing anything blocking.

File: src/components/GetMaps.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  }, [width, height, url]);
+
+  const states = {
+    imgWidth,
+    imgHeight,
+  };
+
+  return {
+    states,
+  };
```
Comment (JosephRivera10) [inline src/components/GetMaps.tsx:47]: Seems like an extra step to return imgWidth and imgHeight.

File: cypress/integration/01-smoke.spec.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { e2e } from '@grafana/e2e';
+
+e2e.scenario({
+  describeName: 'Smoke test',
+  itName: 'Smoke test',
+  scenario: () => {
+    e2e.pages.Home.visit();
+    e2e().contains('Welcome to Grafana').should('be.visible');
+  },
+});
```
Comment (JosephRivera10) [inline cypress/integration/01-smoke.spec.ts:10]: Curious about what this is testing.

Comment (nckswt) [reply cypress/integration/01-smoke.spec.ts:10]: So am I :slightly_smiling_face:

**Pull request merged**